src/screens/Auth/VerifyEmail/VerifyEmail.tsx

```
@@ -0,0 +1,222 @@
+    <LoadingState isLoading={verifyLoading || resendLoading} variant="spinner">
+      <div className={styles.pageWrapper}>
+        <div className="row container-fluid d-flex justify-content-center items-center">
+          <div className="col-12 col-lg-4 px-0">
+            <div className={styles.cardTemplate}>
+              <TalawaLogo className={styles.logo} />
+
+              {verificationState === 'loading' && (
+                <div className={styles.stateContainer}>
+                  <output
+                    className="spinner-border text-primary"
+                    role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)